distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -208,7 +208,11 @@ make_json_for_table dataframe max_rows all_rows_count include_index_col is_db_ta
         number_untrimmed = case all_rows_count > Column.default_sample_size of
             False -> JS_Object.from_pairs [["name", "Count untrimmed whitespace"], ["percentage_value", columns.map .count_untrimmed]]
             True -> JS_Object.from_pairs [["name", "Count untrimmed whitespace (sampled)"], ["percentage_value", columns.map .count_untrimmed]]
-        [number_nothing, number_untrimmed]
+        number_non_triv = case all_rows_count > Column.default_sample_size of
+            False -> JS_Object.from_pairs [["name", "Count non trivial whitespace"], ["percentage_value", columns.map .count_non_trivial_whitespace]]
+            True -> JS_Object.from_pairs [["name", "Count non trivial whitespace (sampled)"], ["percentage_value", columns.map .count_non_trivial_whitespace]]
+        JS_Object.from_pairs 
+        [number_nothing, number_untrimmed, number_non_triv]
```

Comment:
Let's change it so we check the count only once. Tiny nit.

Reply:
Yeah it would be much better if we go the `name_extra` route and avoid the duplication in all the `percentage_value` entries.